contrib_bots/lib/gcalendar.py

```diff
@@ -0,0 +1,333 @@
+from __future__ import print_function
+import httplib2
+import logging
+import os
+import re
+import sys
+
+# Uses Google's Client Library
+#   pip install --upgrade google-api-python-client
+from apiclient import discovery, errors
```

Comment:
If it's to be an expected behavior, we should document it - getting unexpected tracebacks in the console when running the bot may be confusing. Are we using `cache_discovery` actively? If not, why would we want not to disable it, if it can be confusing?